backup_data_tables.wdl

```diff
@@ -0,0 +1,44 @@
+version 1.0
+
+workflow backup_data_tables {
+  input {
+    String workspace_namespace
+    String workspace_name
+    String output_directory
+  }
+  call backup_tables {
+    input: workspace_namespace=workspace_namespace,
+      workspace_name=workspace_name,
+      output_directory=output_directory
+  }
+  output {
+    # Should there actually be output from this workflow if the files are copied to the bucket?
+    Array[File] tables = backup_tables.table_files
+    File json_file = backup_tables.json_file
+  }
+}
+
+
+task backup_tables {
+  input {
+    String workspace_namespace
+    String workspace_name
+    String output_directory
+  }
+  command {
+    set -e
+    Rscript /usr/local/anvil-util-workflows/backup_data_tables.R \
+        --workspace_namespace ~{workspace_namespace} \
+        --workspace_name ~{workspace_name} \
+        --output_directory ~{output_directory}
+  }
+  output {
+    # Should there actually be output from this workflow if the files are copied to the bucket?
+    Array[File] table_files = glob("*.tsv")
+    File json_file = "table_files.json"
+  }
```

Comment:
The tsv files and json file are copied to the specified `output_directory` in the specified workspace. Should the workflow also produce outputs? I am thinking no because:
1. There would be two copies of the files, those in the submission/execution directory and those in the `output_directory`.
2. The json file would only have the paths to the files in the `output_directory`.

Reply:
I agree that the files don't need to be in multiple places